core/embed/rust/src/ui/layout_caesar/component/mod.rs

```diff
@@ -44,7 +44,7 @@ mod share_words;
 mod show_more;
 mod title;
 
-#[cfg(feature = "translations")]
+#[cfg(all(feature = "micropython", feature = "translations"))]
```

Comment:
I think these are needed both before `mod address_details` and `pub use address_details::AddressDetails`?

Reply:
22baa1922a